src/ds/advanced_mode/advanced_mode.cpp

```diff
@@ -753,6 +754,10 @@ namespace librealsense
         get_depth_auto_exposure(&p.depth_auto_exposure);
         get_depth_gain(&p.depth_gain);
         get_depth_auto_white_balance(&p.depth_auto_white_balance);
+        _depth_sensor.finished_bulk_operation();
+
+        if( *_color_sensor )
```

Comment:
Will D421 code get here? with no color sensor?

Reply:
It also has an RAII version no?

Reply:
> Will D421 code get here? with no color sensor?

Yes, that is why we check `*_color_sensor` (which is also checked in the following get_xxx calls.

> It also has an RAII version no?
Yes, I did not use it as I wanted to release power in middle of function. Maybe better to use it in case of an exception. I will update